lib/celluloid/autostart.rb

```diff
@@ -1,5 +1,6 @@
 require "celluloid"
 
+# start up the basic Celluloid services - only necessary to do this manually in old versions
```

Comment:
`only necessary to do this manually in old versions` not sure what you mean by that